services/s3/src/main/java/software/amazon/awssdk/services/s3/internal/crt/CopyObjectHelper.java

```diff
@@ -192,11 +191,10 @@ private static void handleException(CompletableFuture<CopyObjectResponse> return
                                         Throwable throwable) {
         Throwable cause = throwable instanceof CompletionException ? throwable.getCause() : throwable;
 
-        if (cause instanceof SdkException || cause instanceof Error) {
+        if (cause instanceof Error) {
```

Comment:
@joviegas @davidh44 I removed SdkException check here because I found it's much easier for troubleshooting with wrapped exception because the stacktrace is preserved. I deserve a "told you so" 😅